etc/databases/world/updates/updates.sql

```diff
@@ -30418,6 +30418,85 @@ begin not atomic
+        update npc_vendor set slot = 20 where item = 8343 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 21 where item = 4291 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 22 where item = 3713 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 23 where item = 3857 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 24 where item = 3777 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 25 where item = 4341 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 26 where item = 6374 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 27 where item = 2692 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 28 where item = 3466 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 29 where item = 2928 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 30 where item = 8925 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 31 where item = 4400 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 32 where item = 6532 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 33 where item = 3372 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 34 where item = 4399 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+        update npc_vendor set slot = 35 where item = 6530 and entry in(2118, 5163, 4561, 1250, 1286, 5100, 3168, 843, 4897, 4877, 4229, 3614, 3499, 3187, 3482, 66, 4194);
+
+        -- Set the correct subname for Leo Sarn.
+        update creature_template set subname = 'Enchanter' where entry = 5758;
```

Comment:
Please change subname for all `<Enchanting Supplies>` NPCs.

Reply:
I did ... that was in another PR you already merged :) I'll remove the line if need be.

Reply:
> I did ... that was in another PR you already merged :) I've removed that query.

In which PR? I can't find the query in `updates.sql`.

Reply:
Hm, neither can I anymore. Seems to have vanished. I'll readd it.

Reply:
Done

Reply:
Thanks!